src/sheet_id.rs

```diff
@@ -0,0 +1,19 @@
+use std::fmt::{Display, Error as FmtError, Formatter};
```

Comment:
Isn't 
```
pub type SheetId = u64;
```
enough?

Reply:
Having a newtype provides some advantages:

1. The real ID type is encapsulated and becomes an implementation detail
1. We get static validation of ID type, it's impossible to create bug where we pass RowId as SheetId (or any other u64)
1. We remove whole u64 interface, from which 99% of methods are not valid for an ID (multiplication, addition, conversion from u8, etc.)
1. We create new interface, which can have custom, domain specific methods